src/Plugin/VoteUpDownWidget/Alternate.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Drupal\vud\Plugin\VoteUpDownWidget;
+
+use Drupal\vud\Plugin\VoteUpDownWidgetBase;
+
+/**
+ * Plugin "alternate" of type VoteUpDownWidget.
+ *
+ * @VoteUpDownWidget(
```

Comment:
I'll add widget_template with the corresponding name of the widget. But what about vote_template?